epub33/core/index.html

```diff
@@ -448,7 +448,8 @@ <h3>Terminology</h3>
 							in outbound hyperlinks that resolve to <a>Remote Resources</a> (e.g., referenced from the
 								<code>href</code> attribute of an [[!HTML]] <a
 								href="https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element"
-									><code>a</code></a> element).</p>
+									><code>a</code></a> element). Resources represented by <a href="#sec-data-urls">data
+								URLs</a> are also not Publication Resources.</p>
```

Comment:
This seems to allow data URLs as a mechanism to bypass the fallback chain when specifying non-core media types. So an `img` element referencing a file that is not supported would need to provide a fallback, but a data URL would not. And if the aren't Publication Resources, what are they? They aren't foreign resources, since those are defined to be publication resources. I am a little worried about introducing a new type of entity. Is it possible to leave them as publication resources, but remove the requirement to be listed in the manifest in 2.2.2?

Reply:
Ew, ya, that's not what I was intending at all. It was only about an exclusion from the manifest. I'll fix.